loopy/transform/fusion.py

```diff
@@ -419,6 +419,11 @@ def fuse_kernels(programs, suffixes=None, data_flow=None):
         *data_flow* was added in version 2016.2
     """
 
+    from loopy.program import make_program
+
+    programs = [make_program(knl) if isinstance(knl, LoopKernel) else knl for
+            knl in programs]
```

Comment:
After 44d4c497, `fuse_kernels` only takes `LoopKernel`s, this will lead to a some test failures. This is fine because the tests involving fuse_kernels(like [this](https://github.com/inducer/loopy/blob/d3ede00fa60680aa8df487d56fb0549d4582b976/test/test_loopy.py#L1521)) would have to be altered once we concretely define the TranslationUnit interface.